probatus/metric_volatility/volatility.py

```diff
@@ -99,8 +101,9 @@ def __init__(self, model, scoring='roc_auc', test_prc=0.25, n_jobs=1, stats_test
 
         self.stats_tests_objects = []
         if len(self.stats_tests_to_apply) > 0:
-            warnings.warn("Computing statistics for distributions is an experimental feature. While using it, keep in "
-                          "mind that the samples of metrics might be correlated.")
+            if self.verbose>0:
```

Comment:
whitespacing. Might help to run codebase through `flake8` for this kind of stuff?

Reply:
I will make an issue to run `black` on the codebase